torch/_inductor/constant_folding.py

```diff
@@ -253,23 +276,41 @@ def constant_graph_tag(gm: torch.fx.GraphModule):
             node.meta[META_TAG] = MODULE_TAG
 
 
-def run_and_get_constant_graph(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
+def run_and_get_constant_graph(
```

Comment:
Does this supports gm with a mixed of get_attr and lifted constants? 
or do we not expect ppl to have a graph?

Reply:
yeah we support mixture of nodes. the invariant is simpler to specify this way. we should always have a graph.